installation.md

```diff
@@ -69,45 +69,47 @@ should work with any of them, though feature set and interface may vary.
 Here are some plugins we know work well with clangd:
 
 <details>
-<summary markdown="span">Vim</summary>
-[YouCompleteMe](https://valloric.github.io/YouCompleteMe/) can be installed with
-clangd support. **This is not on by default**, you must install it with
-`install.py --clangd-completer`.
+<summary markdown="span">Vim/Neovim</summary>
+We recommend to use [coc.nvim](https://github.com/neoclide/coc.nvim) + [coc-clangd](https://github.com/clangd/coc-clangd)
```

Comment:
I think it's a little confusing that there are two, so maybe explain: "We recommend to use coc.nvim. coc-clangd is a small extension which configures coc.nvim to work with clangd."